src/runtime/opengl.cpp

```diff
@@ -1883,6 +1914,27 @@ WEAK int halide_opengl_device_sync(void *user_context, struct buffer_t *) {
     return 0;
 }
 
+// Copy a string up to the next kernel sentinel.
+// This is used by halide_opengl_initialize_kernels() to cut the
+// full kernel source into per-stage bits
+WEAK bool get_kernel_src(const char *src, char** dst, size_t* len) {
+
+  const char* sentinel = strstr(src+1, kernel_marker);
+  bool done = false;
+  char *ret = (char*)src;
+
+  if (sentinel == NULL) {
+    *len = strlen(src);
+    done = true;
+  }
+  else
```

Comment:
Please use this style:

    if (foo) {
        asd
    } else {
        asd
    }